src/test/scala/Zstd.scala

```diff
@@ -34,6 +34,7 @@ class ZstdSpec extends FlatSpec with Checkers {
       check { input: Array[Byte] =>
         {
           val size        = input.length
+          // Assumes that `Zstd.defaultCompressionLevel() == 3`.
           val compressed  = if (level != 3) Zstd.compress(input, level) else Zstd.compress(input)
```

Comment:
Alternatively we could do `level != Zstd.defaultCompressionLevel()`, but test coverage of `Zstd.compress(input)` would then implicitly rely on `Zstd.defaultCompressionLevel()` being one of the values in the `levels` list. An alternative would be to explicitly add `Zstd.defaultCompressionLevel()` to `levels`, though then _in theory_ the list could contain a duplicate entry.